### ISSUE TITLE
No effect with --convert-primaries displayp3 (v4.3.0-alpha3)

### ISSUE THREAD

**Issue by donmccurdy** [KhronosGroup/KTX-Software#752]
Testing `--convert-primaries displayp3` to convert a source PNG from sRGB to Display P3, the command runs without errors:

```bash
ktx create --format R8G8B8A8_SRGB --zstd 18 --generate-mipmap \
  --assign-oetf srgb --assign-primaries srgb --convert-primaries displayp3  \
  rgba8.png rgba8_displayp3.ktx2
```

However, running `ktx info rgba8_displayp3.ktx2` on the resulting file shows `KHR_DF_PRIMARIES_BT709`. If the conversion isn't implemented yet that's fine — I can certainly convert the source PNG with another tool first. But perhaps an error should be thrown here, if that's the case?

**Comment by MarkCallow** [comment]
This should work afaik. @aqnuep please investigate.

**Comment by aqnuep** [comment]
That's correct. This must have been some recent regression.

It is fixed by PR #753 (and CTS PR https://github.com/KhronosGroup/KTX-Software-CTS/pull/2).

**Comment by donmccurdy** [comment]
Thank you @MarkCallow and @aqnuep!
